apps/molgenis-components/package.json

```diff
@@ -14,7 +14,7 @@
     }
   },
   "scripts": {
-    "dev": "vite",
+    "dev": "cross-env CLEAR_GEN_FOLDERS=on vite",
     "build": "cross-env SHOW_CASE=off vite build",
```

Comment:
we don't need to clear_gen folders on build or build-showcase?
Or is that automatically handled (when added to build.gradle it will be considered build outputs that need cleaned)?

Reply:
yes , i think gradle does a clean all , not sure if doc-gen has been added